### PR TITLE
Improve jobs

### PR DESCRIPTION
**Pull request by RonaldDijkstra**
The return of the BYOD regeling and opleidingsbudget @fatboypunk!

### CODE REVIEW

File: source/jobs/nl/elixir-developer.html.markdown.erb [hunk 1]
```diff
@@ -9,7 +9,7 @@ popup: false
 apply_link: developer
 ---
 
-Wij zijn op zoek naar een nieuwe collega voor het ontwikkelen van onze software. Sinds juni zijn wij bezig met het ontwikkelen van een nieuwe applicatie in [Elixir](https://elixir-lang.org/){:target="_blank"}{:rel="noopener"}{:title="Elixir"} i.c.m. [Phoenix](http://phoenixframework.org/){:target="_blank"}{:rel="noopener"}{:title="Phoenix Framework"}. Deze nieuwe applicatie wordt vanaf de grond af opgebouwd, een mooie kans om aan mee te werken!
+Wij zijn op zoek naar een nieuwe collega voor het ontwikkelen van onze software. Sinds juni zijn wij bezig met het ontwikkelen van een nieuwe applicatie in [Elixir](https://elixir-lang.org/){:target="_blank"}{:rel="noopener"}{:title="Elixir"} i.c.m. [Phoenix](http://phoenixframework.org/){:target="_blank"}{:rel="noopener"}{:title="Phoenix Framework"}. Deze nieuwe applicatie wordt vanaf de grond af opgebouwd, een mooie kans om aan mee te werken! Daarom zijn we op zoek naar een **Elixir Developer**.
```
Comment (fatboypunk) [inline source/jobs/nl/elixir-developer.html.markdown.erb]: why the `elixir developer`?

Comment (RonaldDijkstra) [reply source/jobs/nl/elixir-developer.html.markdown.erb]: I was changing some text layout things, didn't like it, reverted it, but overlooked this

Comment (fatboypunk) [reply source/jobs/nl/elixir-developer.html.markdown.erb]: maybe `een developer die affiniteit heeft met Elixir`?

File: source/localizable/over-ons/_you.erb [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  <% unless root_locale == :en %>
+    <% locale_link_to("/jobs/index.html") do %>
+      <div class="avatar"><%= t("about.team.you") %></div>
+      <h5>
+        <span><%= t("about.team.work_with_us") %></span>
+        <span>Defacto</span>
+      </h5>
+    <% end %>
+  <% end %>
+</li>
```
Comment (fatboypunk) [inline source/localizable/over-ons/_you.erb]: so we do want to show the li but not what's in it?

Comment (RonaldDijkstra) [reply source/localizable/over-ons/_you.erb]: uuh no...

**Pull request merged**